docs/ossm/ossm2-migration/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenShift Service Mesh 2.x migration to 3.x
+
+This document details how to migrate from 2.x to OpenShift Service Mesh 3.x.
+
+## Pre-migration Checklist
+
+Before you begin to migrate your controlplane from OpenShift Service Mesh 2.x to 3.x, ensure you have done the following:
+
+- Update your `ServiceMeshControlPlane` version to the latest OpenShift Service Mesh release.
+- Disable the following features on your `ServiceMeshControlPlane`. These fields are unsupported in 3.0 and must be disabled prior to migration.
+    <!-- TODO: create a separate page for each of these bullet points describing how to migrate off the SMCP managed version. -->
+  - Network Policy management: `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.x `ServiceMeshControlPlane`, you will need to recreate and manage these manually.
+  - Disabled addons:
```

Comment:
> Not sure if we can just reference the blog post here - https://www.redhat.com/en/blog/preparing-openshift-service-mesh-3

That's a great resource and we can probably lift a lot of the information from there but I think it will reduce frustration for users to put as much info as we can into the documentation rather than farming out links to other places.